src/api/CortexApi.ts

```diff
@@ -76,9 +81,15 @@ export interface CortexApi {
     entityRefs: AnyEntityRef[],
   ): Promise<InitiativeActionItem[]>;
 
-  syncEntities(
+  submitEntitySync(
     entities: Entity[],
     customMappings?: CustomMapping[],
-    teamOverrides?: TeamOverrides
-  ): Promise<void>;
+    teamOverrides?: TeamOverrides,
+  ): Promise<EntitySyncProgress>;
+
+  getEntitySyncProgress(): Promise<EntitySyncProgress>;
+
+  getLastEntitySyncTime(): Promise<LastEntitySyncTime>;
```

Comment:
Non-blocking: probably too late for this but I wonder if it would make more sense for this info to come from 1 endpoint.